OrbitQt/orbitglwidget.cpp

```diff
@@ -296,10 +296,12 @@ void OrbitGLWidget::mouseMoveEvent(QMouseEvent* event) {
 }
 
 void OrbitGLWidget::enterEvent(QEvent* event) {
+  UNUSED(event);
```

Comment:
in which case you won't need this UNUSED here.

Reply:
have you decided to go with UNUSED?

Reply:
We have kind of agreed at some point that we don't use UNUSED anymore (and just omit the name).

Reply:
I found the UNUSED more verbose, but I've changed it now and removed the parameter name for consistency